.github/workflows/tauri-ci.yml

```
@@ -0,0 +1,420 @@
+          cp .env.example ~/.config/Dash-Evo-Tool/.env 2>/dev/null || true
+
+      - name: Build frontend
+        run: npm run build
+
+      - name: Build Tauri app (debug)
+        working-directory: src-tauri
+        run: cargo build 2>&1 | tail -5
+
+      - name: Run WebdriverIO E2E tests
+        timeout-minutes: 25
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2046:warning:2:6: Quote this to prevent word splitting

(shellcheck)